.eleventy.js

```diff
@@ -49,6 +50,7 @@ const YouTube = require(`./${componentsDir}/YouTube`);
 
 const collectionsDir = 'src/site/_collections';
 const authors = require(`./${collectionsDir}/authors`);
+const eventSchedule = require(`./${collectionsDir}/event-schedule`);
```

Comment:
This doesn't need to be an eleventy collection. You can access items in the `_data` folder inside of any page template by using its directory name. For example, the `event.json` file is available to all pages as `{{ event }}`.

https://www.11ty.dev/docs/data-global/